redirects-transition.conf

```diff
@@ -1,5 +1,13 @@
 #Redirect rules for specific pages
 
+rewrite ^/eeo/LimitedEnglishProficiencyPlan.shtml https://www.fec.gov/about/equal-employment-opportunity/limited-english-proficiency/ redirect;
+rewrite ^/eeo/eeo.shtml https://www.fec.gov/about/equal-employment-opportunity-/ redirect; 
+rewrite ^/about/offices/EEO/EEO.shtml https://www.fec.gov/about/equal-employment-opportunity-/ redirect;
```

Comment:
Redirect URL here should be: https://www.fec.gov/about/equal-employment-opportunity/